themes/default/content/docs/concepts/resources/names.md

```diff
@@ -141,7 +141,7 @@ let role = new aws.iam.Role("my-role", {
 
 ```python
 role = iam.Role('my-role', {
-    name='my-role-001'
+    'name': 'my-role-001'
```

Comment:
Hi @SnoozeFreddo,

Thank you for your submission! I actually believe the issue here is the use of the curly braces. This should actually be:

```python
role = aws.iam.Role('my-role', 
    name='my-role-001'
)
```

Reply:
I just assumed that the internal *kwargs option will somehow magically work. I also don't have aws, i therefore couldn't check if it's working. 

Anyway, i just pushed changed to put it in the format you mentioned and used 'black' on it.